src/maintainer/knowledge_base.rst

```diff
@@ -118,28 +118,55 @@ use a general web search to explore — while these topics are beyond the
 scope of this documentation, there are ample discussions on them on the broader
 Internet.
 
+To bootstrap a conda environment and install ``conda-build``, consider
+`miniforge<https://github.com/conda-forge/miniforge>`__.
+
+
+Executing a build
+^^^^^^^^^^^^^^^^^
+
+The ``build-locally.py`` script does not support Windows. Use ``conda build
+recipe/``.
+
+Using a local channel looks a bit different in Windows, too (note the
+triple-forward-slash in the channel name):
+
+.. code-block::
+
+    conda create -n my-new-env -c file:///C:\Users\User\miniforge3\conda-build my-package
+
+If you installed from a different distribution than ``miniforge`` or to a
+different path, you'll need to alter this path.
+
+
+Notes on native code
```

Comment:
I didn't have to deal with native code to build my package today. I was distracted by the many sections on dealing with native code, so I felt they'd be better off together in one section.